Libraries/Animated/src/Interpolation.js

```diff
@@ -163,6 +165,16 @@ function interpolate(
   return result;
 }
 
+function colorToRgba(
+  input: string
+): string {
+  var color = tinycolor(input);
+  var {r, g, b, a} = color.toRgb();
```

Comment:
Calling toRBG() on an invalid color is not safe in the sense that you're going to get an unexpected result. Either a crash or some arbitrary value.

In this case you know what it does because you dug in tinycolor implementation to figure it out. But the next person that's going to modify this function will see that r, g and b are defined and will use them even though it will not make sense.

In isolation this isn't dramatic, but as your codebase grows and there are unsafe accesses like this all over this is going to cause real bugs and worse, people are going to do crazy hacks to work around that.

A not too far fetched example would be that you somehow return rbg without the isvalid information in a function that's widely used. I can bet money that there will be some code that checks if rgb is black and then find the original string and do === 'black' to make sure the user really put black and not some random variable.